packages/components/src/components/wallet-icon/wallet-icon.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { CSSProperties } from 'react';
+import Icon from '../icon';
+import { GradientBackgroundTwoPoint } from '../gradient-background-two-point';
+import './wallet-icon.scss';
+import classNames from 'classnames';
+
+type TWalletIcon = {
+    icon: string;
+    iconSize: number | string;
+    primaryColor: CSSProperties['backgroundColor'];
+    secondaryColor: CSSProperties['backgroundColor'];
+    rounded?: string;
+    size: 'small' | 'medium' | 'large';
+};
+
+const getBlurRadius = (size: string) => {
+    if (size === 'small') return 16;
+    if (size === 'medium') return 24;
+
+    return 48;
+};
+
+const WalletIcon = ({ icon, iconSize = 24, primaryColor, secondaryColor, rounded, size = 'medium' }: TWalletIcon) => {
+    return (
+        <div className={`wallet-icon wallet-icon--${size}`}>
+            <div className='wallet-icon__background'>
```

Comment:
@aum-deriv The `GradientBackgroundTwoPoint` component already is accepting children, You can pass the `Icon` as a child to it so you won't need the extra style for `wallet-icon__background` and `wallet-icon__icon`.

```tsx
<div className={`wallet-icon wallet-icon--${size}`}>
    <GradientBackgroundTwoPoint
        blurRadius={getBlurRadius(size)}
        backgroundColor='var(--general-main-2)'
        primaryColor={primaryColor}
        secondaryColor={secondaryColor}
    >
        <Icon icon={icon} size={iconSize} />
    </GradientBackgroundTwoPoint>
</div>
```